lightning/src/ln/channelmanager.rs

```diff
@@ -3034,7 +3063,7 @@ where
 		if chan.context.get_short_channel_id().is_none() {
 			return Err(LightningError{err: "Channel not yet established".to_owned(), action: msgs::ErrorAction::IgnoreError});
 		}
-		log_trace!(self.logger, "Attempting to generate broadcast channel update for channel {}", log_bytes!(chan.context.channel_id()));
+		log_trace!(WithChannelContext::from(&self.logger, &chan.context), "Attempting to generate broadcast channel update for channel {}", log_bytes!(chan.context.channel_id()));
```

Comment:
IMO, I actually think this does currently reads quite well. `log_trace!(withChannelContext...`
We could do `WithChannel` instead of `WithChannelContext` to reduce it further (because no one actually needs to understand what `ChannelContext` means, to them a channel is a channel)